website/src/content/main-concepts/proxy-to-external-api.mdx

```diff
@@ -214,6 +214,32 @@ exports.handler = async function (request, response) {
 };
 ```
 
+### Decoding the token
+
+You can decode the JWT token with [JWT.io](https://jwt.io/). In the decoded version, you will see the following fields.
+
+```json
+{
+  "exp": 1600092401,
+  "sub": "<subject>",
+  "iss": "<hostname>",
+  "aud": "<audience-url>",
+  "type": "exchange",
+  "<hostname>/claims/project_key": "<project-key>",
+  "iat": 1600092341
+}
+```
+
+These fields are according to specification **Register Claim Names** and consist of the following:
+
+- `exp`: This claim identifies the **expiration time**. The claims requires to be before the current time in order to accepted for processing.
+- `sub`: This claim identifies the **subject**. The value of `sub` issued by the API Gateway for Custom Applications is the user-ID for Merchant Center.
+- `iss`: This claim identifies the **issuer**. The value of `iss` is the hostname to the API Gateway itself. This value differs depending on the region the JWT was issued. These are listed under API Gateway under **Hostnames**.
```

Comment:
I couldn't link to the **API Gateway page -> Hostnames**

Reply:
never mind. we already outlined **cloud regions** and **MC API url** above, so I point to that.